packages/amplify-console-integration-tests/src/util.ts

```diff
@@ -2,6 +2,8 @@ import * as path from 'path';
 import * as fs from 'fs-extra';
 import * as moment from 'moment';
 
+export { getCLIPath, isCI, npmInstall } from 'amplify-e2e-core';
+
 export function createNewProjectDir(mark?: string): string {
```

Comment:
We could reuse this by moving it to e2e core and have parity as how we name our tests. Just naming them `console-inge` would make it harder to track which test is is not cleaning up the resources, when the resources remain in the account.  We also have had issues with test name collisions as the granularity of the  test name is seconds and not milli or nano seconds.
https://github.com/aws-amplify/amplify-cli/blob/c6ea964712d1fc127466822638a076c7492260ab/packages/amplify-e2e-tests/src/utils/index.ts#L30-L42